### DISCUSSION TITLE
scale out computing support schedule?

### DISCUSSION THREAD

**Discussion by dudu231** [secretflow/secretflow#113] [Q&A]
when will this secretflow support multi computation node scaling out as it claimed?

**Comment by 6fj** [comment]
Hi @dudu231 

Could you please clarify your requirement?

- **computation node**, is this for PYU devices or Parties of SPU device?
- **scaling out**, could you please describe the scenarios when the performance of lastest version of SecretFlow couldn't meet your requirements?

Thanks!
